features/attached_enable.feature

```diff
@@ -795,6 +795,12 @@ Feature: Enable command behaviour when attached to an UA subscription
         ROS ESM Security Updates cannot be enabled with UA Apps: ESM disabled.
         Enable UA Apps: ESM and proceed to enable ROS ESM Security Updates\? \(y\/N\) Cannot enable ROS ESM Security Updates when UA Apps: ESM is disabled.
         """
+        When I verify that running `ua enable ros --beta --format json` `with sudo` and stdin `N` exits `1`
+        Then I will see the following on stdout:
+        """
+        ROS ESM Security Updates cannot be enabled with UA Apps: ESM disabled.
```

Comment:
I think when we have `--format json` we should have only JSON returned in the screen. The prompt messages need to be aware of that somehow.
If any script is calling this and trying to parse the JSON response, it will probably error out on the prompt line.

I am just not sure of how to handle this:
- do we require `--assume-yes` when enabling services which depend on others, and error out in the JSON if it is not passed?
- do we `--assume-yes` all the time and forget about the prompt?

Reply:
I think we should require that `--assume-yes` be passed at the cli if `--format=json` is passed.
So `ua enable ros --beta --format=json` would error immediately saying something like:
`{ "error": [{"message": "json formatted response requires --assume-yes flag"}]}`

Then we won't have this problem, and I think it makes sense from a user perspective.

Reply:
That's a good question, but I don't know how to better handle. My original idea was to use `--assume-yes` by default when running a command, but we discussed on #1912 and we decided not to use it.

My current opinion is that an user will be able to understand that it needs the `--assume-yes` if the command is running in an automated fashion. 

But maybe we could use the first idea, even though it make the command a little bit more verbose. What do you think @orndorffgrant ?

Reply:
And I just realized my suggestion is exactly @renanrodrigo's first suggestion, so that is what I think we should do  :)

Reply:
Actually, my suggestion is a little stronger. I think we should always require --assume-yes if --format=json (not only if the service has dependent services).
This is simpler to understand and implement, plus it is more consistent for the user - it doesn't require them to understand which services may require a dependent service or not when determining if they need the flag or not

Reply:
Okay, I will make that flag required then on `--format json`